src/main/java/com/kodilla/ecommercee/entity/UserOrder.java

```diff
@@ -19,4 +21,17 @@
     @Id
     @GeneratedValue
     private Long Id;
+
+    private Date orderMade;
+
+    private Boolean resolved;
+
+    @OneToOne(cascade = CascadeType.ALL,
+            fetch = FetchType.EAGER)
+    @JoinColumn
+    private GenericEntity user;
```

Comment:
Tutaj powinna być relacja do encji User a nie GenericEntity. Jeśli takiej nie ma to utwórz sama encji mającą tylko Id, a ktoś ją potem rozbuduje